salt/beacons/status.py

```diff
@@ -118,7 +127,12 @@ def beacon(config):
 
     ret = {}
     for func in config:
-        data = __salt__['status.{0}'.format(func)]()
+        try:
+            data = __salt__['status.{0}'.format(func)]()
+        except salt.exceptions.CommandExecutionError as exc:
+            log.error('Status beacon attempted to process function {0} \
```

Comment:
Lets change this to a debug, otherwise it will just fill up the logs on windows etc.